### PR TITLE
Add "Open in Colab" badges to all notebooks

### PR DESCRIPTION
**Pull request by adamnsch**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [x] Your contribution is covered by tests

### CODE REVIEW

File: doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ predicting the genre of `Movie` nodes
 * Train the pipeline with autotuning and inspecting the results
 * Make predictions for movie nodes missing without a specified genre
 
+
+https://colab.research.google.com/github/neo4j/graph-data-science-client/blob/main/examples/heterogeneous-node-classification-with-hashgnn.ipynb[image:https://colab.research.google.com/assets/colab-badge.svg[Open
+In Colab]]
```
Comment (FlorentinD) [inline doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: In previous notebooks we added the badge at the very top of the notebook, even before the intro.

I think we should align the place. Right now, 2 have the badge at the top and the new 4 badges after the intro

**Review by FlorentinD — APPROVED** [review]
Awesome :bee:

File: doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc (hunk 1 above)
Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: Yeah I guess :) @nvitucci want to weigh in on this?

Comment (nvitucci) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: Personally I think it looks better _after_ the title. Besides causing conversion issues when appearing on top (that we could probably solve), to me it looks like a better positioning as it's an "implementation detail" once the topic has been introduced. Do you have any objections to this?

Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: Not sure I understand what you mean. I think we all agree that it should be after the title :) The question was whether the badge should be after title + introduction test or just after the title. Hope that makes sense

Comment (nvitucci) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: Ah, sorry, I misunderstood that. I'd propose to keep it right after the title, followed by everything else.

Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc:16]: Ok! Thanks for the info :) Then I'll need to make some changes

Comment (adamnsch) [reply doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc]: Done!

File: doc/README.md [hunk 2]
```diff
@@ -121,12 +121,24 @@ The script `../scripts/nb2doc/convert.sh` can be used:
...
+./scripts/nb2doc/convert.sh
 ```
 
+
 ### Style notes
 
 For a successful conversion of the notebooks, some style notes apply.
 
 * A notebook must only contain one first-level header, which should be in the first cell (as a title).
+* The notebook should contain an "Open with Colab" badge after its title (instructions below)
```
Comment (nvitucci) [inline doc/README.md:133]:
```suggestion
* The notebook should contain an "Open with Colab" badge after its title and before any other section (instructions below)
```

Comment (adamnsch) [reply doc/README.md]: done

**Review by nvitucci — APPROVED** [review]

**Pull request merged**